device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceTwin/DeviceMethod.java

```diff
@@ -15,7 +15,7 @@
     private Object deviceMethodCallbackContext;
     private final IotHubEventCallback deviceMethodStatusCallback;
     private final Object deviceMethodStatusCallbackContext;
-    private final ObjectLock DEVICE_METHOD_LOCK = new ObjectLock();
```

Comment:
ObjectLock was just a reinvention of the Object class, and has been deleted

Reply:
look at us, reinventing the wheel :D